packages/api-client/src/createAPIClient.ts

```diff
@@ -154,13 +154,23 @@ export function createAPIClient<
       ...(currentParams.fetchOptions || {}),
     };
 
+    let mergedHeaders = defu(currentParams.headers, defaultHeaders);
```

Comment:
the interesting line here: `defu` accepts the defaults as a **last** parameter. lesson learnt 😂

Reply:
Yeah, there is no consequent style that works for all languages and tools.
I always look this up ... so much wasted time :joy: